wagtail/admin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -186,10 +185,8 @@ footer {
         }
     }
 
-    $width-1440px: 90em;
-
-    @media screen and (min-width: $width-1440px) {
-        width: $width-1440px;
+    @include media-breakpoint-up(xl) {
+        width: breakpoint-min(xl);
```

Comment:
This seems to be the only place where the `xl` breakpoint is used. Should we re-do this with either `lg` or `xxl`?

Reply:
Ironic: it turns out it's not even called since the following is since the following from `sm` get applied.

![image](https://user-images.githubusercontent.com/287422/47905515-a1204d00-de87-11e8-9a8c-41bc4660a0dd.png)